src/ControlSystem/Tags/QueueTags.hpp

```diff
@@ -33,4 +35,59 @@ template <typename Frame>
 struct Horizon {
   using type = ::Strahlkorper<Frame>;
 };
+
+/// \ingroup ControlSystemGroup
+/// Holds a full strahlkorper from measurements for the excision surface
+template <typename Frame>
+struct ExcisionSurface {
+  using type = ::Strahlkorper<Frame>;
+};
+
+/// \ingroup ControlSystemGroup
+/// Holds a full strahlkorper from measurements for the time derivative of the
+/// apparent horizon surface
+template <typename Frame>
+struct TimeDerivApparentHorizon {
+  using type = ::Strahlkorper<Frame>;
+};
+
+/// \ingroup ControlSystemGroup
+/// Holds the lapse on the `ExcisionSurface`
+struct LapseOnExcisionSurface {
+  using type = Scalar<DataVector>;
+};
+
+/*!
+ * \ingroup ControlSystemGroup
+ * \brief Holds a quantity that's similar to the shift, but isn't the shift, on
+ * the `ExcisionSurface`.
+ *
+ * \details This holds
+ *
+ * \f{equation}{
+ * \beta^i \frac{\partial x^\hat{i}}{\partial x^i} =
+ * \hat{beta}^\hat{i} + \frac{\partial x^\hat{i}}{\partial t}
+ * \f}
+ *
+ * where hatted quantities are in the distorted frame and non-hatted quantities
+ * are in the grid frame.
+ */
+template <typename Frame>
+struct LetsGetShifty {
```

Comment:
I'll allow you some freedom in naming, but I will insist that you at least use a noun.

Reply:
lol that's fair